.github/workflows/test-environment.yml

```diff
@@ -142,6 +147,22 @@ jobs:
           aws s3 cp "${{ env.EC2_CSPM_KEY }}" "${{ env.S3_BUCKET }}/cspm.pem"
           aws s3 cp "${{ env.EC2_KSPM_KEY }}" "${{ env.S3_BUCKET }}/kspm.pem"
 
+      - name: Summary
+        if: always()
```

Comment:
if previous steps failed, will this step have any meaning?

Reply:
I agree, it can be replaced by `success()`.